### PR TITLE
Prune tests for features that are no longer supported.

### PR DESCRIPTION
**Pull request by jcoyne**
Fix FileSet#parents
Remove unused FileSet::Querying

Added a draft of release notes

### CODE REVIEW

File: app/models/concerns/hyrax/file_set/belongs_to_works.rb [hunk 1]
```diff
@@ -7,8 +7,10 @@ module BelongsToWorks
       #   before_destroy :remove_representative_relationship
       # end
 
+      # Objects that call this FileSet a member
       def parents
-        in_works
+        query_service = Valkyrie::MetadataAdapter.find(:indexing_persister).query_service
+        query_service.find_inverse_references_by(resource: self, property: :member_ids)
```
Comment (cjcolvar) [inline app/models/concerns/hyrax/file_set/belongs_to_works.rb]: Could this be simplified to `query_service.parents(resource: self)`?  If so, do we even need this method/module at all?

File: spec/models/file_set_spec.rb [hunk 2]
```diff
@@ -205,93 +137,9 @@ class AltFile < ActiveFedora::Base
...
-        m.apply_depositor_metadata('jcoyne')
-        m.permissions_attributes = [{ type: 'person', access: 'read', name: 'person1' },
-                                    { type: 'person', access: 'read', name: 'person2' },
-                                    { type: 'group', access: 'read', name: 'group-6' },
-                                    { type: 'group', access: 'read', name: 'group-7' },
-                                    { type: 'group', access: 'edit', name: 'group-8' }]
-      end
-    end
-
-    it 'has read groups accessor' do
```
Comment (cjcolvar) [inline spec/models/file_set_spec.rb]: Did you mean to remove this test?

Comment (jcoyne) [reply spec/models/file_set_spec.rb]: Yeah, this is a method on Valkyrie::Resource, not on this class.

File: app/models/concerns/hyrax/file_set/belongs_to_works.rb (hunk 1 above)
Comment (jcoyne) [reply app/models/concerns/hyrax/file_set/belongs_to_works.rb]: I've made the suggested change and deprecated this method.

File: spec/models/file_set_spec.rb [hunk 3]
```diff
@@ -462,6 +192,7 @@ def paranoid_edit_permissions
     end
   end
 
+<<<<<<< HEAD
```
Comment (cjcolvar) [inline spec/models/file_set_spec.rb]: Looks like this get left here.

File: spec/models/file_set_spec.rb [hunk 4]
```diff
@@ -509,6 +240,8 @@ def paranoid_edit_permissions
     end
   end
 
+=======
+>>>>>>> Prune tests for features that are no longer supported.
```
Comment (cjcolvar) [inline spec/models/file_set_spec.rb]: Looks like this get left here.

**Pull request merged**